aime.aime

```diff
@@ -0,0 +1,7 @@
+integer
+main(void)
+{
+    o_text("Hello World\n");
+ 
+    return 0;
+}
```

Comment:
From my findings, it seems like you'd only need this:
```aime
o_text("Hello world!\n");
```
Is there any reason to add the extra code in that you have?

Reply:
Yes, this can also be written. And produce the same output

Reply:
Alright, I can't particularly test this, so I'm trusting you here.